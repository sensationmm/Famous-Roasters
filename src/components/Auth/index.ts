import { Authenticator } from './Authenticator'
import { AuthFormEmail } from './AuthFormEmail'
import { AuthFormItemInput } from './AuthFormItemInput'
import { AuthFormPassword } from './AuthFormPassword'
import { AuthSignIn } from './SignIn'

export { Authenticator as Authenticator }
export { AuthFormEmail as AuthFormEmail }
export { AuthFormItemInput as AuthFormItemInput }
export { AuthFormPassword as AuthFormPassword }
export { AuthSignIn as AuthSignIn }
