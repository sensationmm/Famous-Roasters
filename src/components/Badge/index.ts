import { Badge } from './Badge'

export { Badge as Badge }
