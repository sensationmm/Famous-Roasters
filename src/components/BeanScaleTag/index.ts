import { BeanScaleTag } from './BeanScaleTag'

export { BeanScaleTag as BeanScaleTag }
