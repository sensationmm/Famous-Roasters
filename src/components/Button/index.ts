import { Button, Color, Emphasis, Size } from './Button'

export { Button as Button }
export { Color as ButtonColor }
export { Emphasis as ButtonEmphasis }
export { Size as ButtonSize }
