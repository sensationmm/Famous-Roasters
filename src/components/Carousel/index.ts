import { Carousel } from './Carousel'

export { Carousel as Carousel }
