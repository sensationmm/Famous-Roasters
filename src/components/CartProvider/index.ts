import { CartContext, CartProvider } from './CartProvider'

export { CartContext as CartContext }
export { CartProvider as CartProvider }
