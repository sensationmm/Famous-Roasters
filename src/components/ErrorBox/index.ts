import { ErrorBox } from './ErrorBox'

export { ErrorBox }
