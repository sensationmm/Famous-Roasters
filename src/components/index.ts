import { Button, ButtonColor, ButtonEmphasis, ButtonSize } from './Button'
import { Footer } from './Footer'
import { Layout } from './Layout'
import { Navigation } from './Navigation'
import { Typography, TypographySize, TypographyType } from './Typography'

export { Button as Button }
export { ButtonColor as ButtonColor }
export { ButtonEmphasis as ButtonEmphasis }
export { ButtonSize as ButtonSize }
export { Footer as Footer }
export { Layout as Layout }
export { Navigation as Navigation }
export { Typography as Typography }
export { TypographyType as TypographyType }
export { TypographySize as TypographySize }
