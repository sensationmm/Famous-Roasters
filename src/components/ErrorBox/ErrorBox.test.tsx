import { render, waitFor } from '@testing-library/react'
import React from 'react'
import { Typography } from 'src/components'

import { ErrorBox } from '.'

describe('ErrorBox component', () => {
  it('Renders correctly', async () => {
    const { container } = render(
      <ErrorBox>
        <Typography>Text goes here</Typography>
      </ErrorBox>,
    )
    await waitFor(() => new Promise((res) => setTimeout(res, 0)))
    expect(container).toMatchSnapshot()
  })

  it('Renders correctly with className override', async () => {
    const { container } = render(
      <ErrorBox className="text-bold">
        <Typography>Text goes here</Typography>
      </ErrorBox>,
    )
    await waitFor(() => new Promise((res) => setTimeout(res, 0)))
    expect(container).toMatchSnapshot()
  })
})
