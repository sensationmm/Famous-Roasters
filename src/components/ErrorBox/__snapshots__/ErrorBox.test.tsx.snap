// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBox component Renders correctly 1`] = `
<div>
  <div
    class=" flex bg-brand-grey-whisper p-4 mb-8 rounded"
  >
    <svg
      class="mt-1 mr-3 fill-negative shrink-0"
      fill="inherit"
      height="16"
      viewBox="0 0 16 16"
      width="16"
    >
      <path
        clip-rule="evenodd"
        d="M8 16C12.4183 16 16 12.4183 16 8C16 3.58172 12.4183 0 8 0C3.58172 0 0 3.58172 0 8C0 12.4183 3.58172 16 8 16ZM6.70711 5.29289C6.31658 4.90237 5.68342 4.90237 5.29289 5.29289C4.90237 5.68342 4.90237 6.31658 5.29289 6.70711L6.58579 8L5.29289 9.29289C4.90237 9.68342 4.90237 10.3166 5.29289 10.7071C5.68342 11.0976 6.31658 11.0976 6.70711 10.7071L8 9.41421L9.29289 10.7071C9.68342 11.0976 10.3166 11.0976 10.7071 10.7071C11.0976 10.3166 11.0976 9.68342 10.7071 9.29289L9.41421 8L10.7071 6.70711C11.0976 6.31658 11.0976 5.68342 10.7071 5.29289C10.3166 4.90237 9.68342 4.90237 9.29289 5.29289L8 6.58579L6.70711 5.29289Z"
        fill="currentColor"
        fill-rule="evenodd"
        stroke="none"
        stroke-width="none"
      />
    </svg>
    <span
      class="text-sm leading-5"
    >
      <span
        class="text-base leading-6"
      >
        Text goes here
      </span>
    </span>
  </div>
</div>
`;

exports[`ErrorBox component Renders correctly with className override 1`] = `
<div>
  <div
    class="text-bold flex bg-brand-grey-whisper p-4 mb-8 rounded"
  >
    <svg
      class="mt-1 mr-3 fill-negative shrink-0"
      fill="inherit"
      height="16"
      viewBox="0 0 16 16"
      width="16"
    >
      <path
        clip-rule="evenodd"
        d="M8 16C12.4183 16 16 12.4183 16 8C16 3.58172 12.4183 0 8 0C3.58172 0 0 3.58172 0 8C0 12.4183 3.58172 16 8 16ZM6.70711 5.29289C6.31658 4.90237 5.68342 4.90237 5.29289 5.29289C4.90237 5.68342 4.90237 6.31658 5.29289 6.70711L6.58579 8L5.29289 9.29289C4.90237 9.68342 4.90237 10.3166 5.29289 10.7071C5.68342 11.0976 6.31658 11.0976 6.70711 10.7071L8 9.41421L9.29289 10.7071C9.68342 11.0976 10.3166 11.0976 10.7071 10.7071C11.0976 10.3166 11.0976 9.68342 10.7071 9.29289L9.41421 8L10.7071 6.70711C11.0976 6.31658 11.0976 5.68342 10.7071 5.29289C10.3166 4.90237 9.68342 4.90237 9.29289 5.29289L8 6.58579L6.70711 5.29289Z"
        fill="currentColor"
        fill-rule="evenodd"
        stroke="none"
        stroke-width="none"
      />
    </svg>
    <span
      class="text-sm leading-5"
    >
      <span
        class="text-base leading-6"
      >
        Text goes here
      </span>
    </span>
  </div>
</div>
`;
